docs/foundations/typography/choosing-type/index.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Choosing type
+heading: Typography
+sidenavTitle: Typography
+layout: layouts/pages/has-toc.njk
+permalink: /foundations/typography/choosing-type/index.html
+tags:
+  - typography
+subnav:
+  collection: sortedTypography
+  order: 3
+---
+
+## Overview
+
+We use 3 categories of type: Expressive, Editorial, and Utility. Each category is designed for a specific use case.
```

Comment:
Can we link these 3 types to their respective sections below? I think it'll help with affordance and understanding, especially since this "list" is in a paragraph rather than an actual list.

With that, if we wanted...lists are easier to scan (but we'd have to apply that to the other pages too, like Font Family).

<img width="803" alt="image" src="https://github.com/user-attachments/assets/f32544f9-7bfd-483b-882b-1bc58e8bb73e" />

Reply:
I added links but kept the sentence format the same since there are jump links at the top of the page.